browser/ui/webui/brave_webui_source.cc

```diff
@@ -324,6 +328,15 @@ void CustomizeWebUIHTMLSource(const std::string &name,
         { "geminiWidgetUnitPrice", IDS_GEMINI_WIDGET_UNIT_PRICE },
         { "geminiWidgetTotalPrice", IDS_GEMINI_WIDGET_TOTAL_PRICE },
         { "geminiWidgetTotalAmount", IDS_GEMINI_WIDGET_TOTAL_AMOUNT },
+#if BUILDFLAG(MOONPAY_ENABLED)
```

Comment:
@petemill seems like we should look at splitting these out and maybe have methods like moonpay::CustomizeHTMLSource, etc.. wdyt? @ryanml nothing that needs to be done for this PR

Reply:
++ on this one

Reply:
yes I don't know why we have this brave_webui_source file at all. Each webui should define its own resources and roots.

Reply:
it started with just one webui lol

Reply:
then became a dumping ground